datastore-api/tests/conftest.py

```diff
@@ -90,6 +90,7 @@ def test_node() -> Document:
     return Document(
         __root__={
             "id": "n111",
+            "name": "obama",
```

Comment:
These are unrelated to this PR. Please remove them and create a new PR if needed